api/user/user.ts

```diff
@@ -0,0 +1,45 @@
+export class UserModel {
+  data: string
+  'default-service': string
+  'default-url': string
+  exp: number
+  forename: string
+  group: string
+  iat: number
+  id: string
+  jti: string
+  loa: number
+  sub: string
+  surname: string
+  type: string
+  constructor(prop) {
+    Object.assign(this, prop)
+  }
+}
+export class User {
```

Comment:
what is the difference between usermodel and user? can we name them to have more sense

Reply:
I was just refactoring and didn't look at the naming convention, but 'name' + Model is a good approach